daemon/algod/api/server/v2/errors.go

```diff
@@ -0,0 +1,30 @@
+package v2
+
+var (
```

Comment:
I need to take a pass through the errors and make sure I copied them all over from v1 correctly, a lot of these aren't used and will either go away or the code will be updated to use them.